From ee692ab89f1321ccfb24316c30fce5a90e6a9764 Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Mon, 28 Apr 2014 09:51:42 +0200
Subject: [PATCH] pipelimit: run timer at 1000ms

- proper execution on timer_interval
- reported by Julia <juliabo@gmail.com>

(cherry picked from commit f9d95734ba7c04a0188077914161395c2e94f3eb)
---
 modules/pipelimit/pipelimit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/pipelimit/pipelimit.c b/modules/pipelimit/pipelimit.c
index 77d2796..5aeb071 100644
--- a/modules/pipelimit/pipelimit.c
+++ b/modules/pipelimit/pipelimit.c
@@ -308,7 +308,7 @@ static int mod_init(void)
 		return -1;
 	}
 	timer_init(pl_timer, pl_timer_handle, 0, F_TIMER_FAST);
-	timer_add(pl_timer, MS_TO_TICKS(1500)); /* Start it after 1500ms */
+	timer_add(pl_timer, MS_TO_TICKS(1000)); /* Start it after 1000ms */
 
 	/* bind the SL API */
 	if (sl_load_api(&slb)!=0) {
-- 
2.0.0.rc0

